MAINTAINING.md

```diff
@@ -119,19 +119,30 @@ complete that no input from the original developer or maintainer is required.
 
 ### Organisation of reviewing between maintainers
 
-This section is a placeholder for further discussion around making the review
-process more efficient and effective when multiple maintainers are involved.
-This could include:
+#### Partial review
 
--   Usage of labels
--   Division of review responsibilities, e.g. with ACKs for different areas of
-    review
--   Usage of GitHub functionality, for example "Reviewers" and "Assignees" lists
+When you review a PR only partly according to the points outlined in the
+[technical guidelines](#technical-guidelines) please do not "approve" the PR to
+prevent accidental merges. Rather, give your a verbal ACK and describe what you
+reviewed. In addition, if you processed or reasonably stepped over a whole
+section, mark the PR with the according label from the "Reviewed:" category.
+If you disagree with the assessment of a previous review, you may remove a
+certain "Reviewed:" label.
+
+When all "Reviewed:" labels are set you may give your approval for the PR.
 
 #### Github etiquette
 
--   If there are multiple maintainers reviewing a PR, always give the other
-    maintainers reasonable time to ACK before dismissing their review.
+It is good etiquette to describe what you reviewed, even if you gave the PR a
+full review and gave your approval. This way the contributor and other
+maintainers are able to follow your thought process.
+
+Only assign yourself to PRs, you can however request reviews from other
```

Comment:
Two sentences here. The first is "Only assign yourself to PRs."

Reply:
"Only assign yourself to PRs" is also ambigous. Only assign *yourself* to PRs, or only assign yourself to *PRs*?

Reply:
> "Only assign yourself to PRs" is also ambigous. Only assign *yourself* to PRs, or only assign yourself to *PRs*?

I don't really get, what you are trying to say...

Reply:
> I don't really get, what you are trying to say...

Me neither, but maybe it was a joke and @kaspar030 wanted to say "Only assign yourself t**w**o PRs" :laughing:

Reply:
I think what @kaspar030 is trying to say is that it could be interpreted as:
 1. Maintainers are only allowed to assign themselves to PRs and not allowed to assign other maintainers
 2. Maintainers are only allowed to assign themselves to PRs and not to other things (e.g. issues) that they could assign themselves to.

Reply:
Ahhhhh ^^ Sorry then it was just my poor English (or rather post-lunch brain) that confused *ambiguous* with *redundant* ^^"